app/routers/profile.py

```diff
@@ -1,15 +1,16 @@
-import io
-
-from fastapi import APIRouter, Depends, File, Request, UploadFile
-from starlette.responses import RedirectResponse
 from starlette.status import HTTP_302_FOUND
```

Comment:
Sort alphabetically, move io to be in the first paragraph and app.internal to be in the third one